x-pack/plugins/apm/server/routes/fallback_to_transactions/route.ts

```diff
@@ -8,13 +8,21 @@
 import * as t from 'io-ts';
 import { getIsUsingTransactionEvents } from '../../lib/helpers/transactions/get_is_using_transaction_events';
 import { createApmServerRoute } from '../apm_routes/create_apm_server_route';
-import { kueryRt, rangeRt } from '../default_api_types';
+import {
+  kueryRt,
+  rangeRt,
+  serviceTransactionDataSourceRt,
+} from '../default_api_types';
 import { getApmEventClient } from '../../lib/helpers/get_apm_event_client';
 
 const fallbackToTransactionsRoute = createApmServerRoute({
   endpoint: 'GET /internal/apm/fallback_to_transactions',
   params: t.partial({
-    query: t.intersection([kueryRt, t.partial(rangeRt.props)]),
+    query: t.intersection([
+      kueryRt,
+      t.partial(rangeRt.props),
+      serviceTransactionDataSourceRt,
```

Comment:
This seems redundant as its not being used ?
Same applies to on the frontend

Reply:
Good catch, a leftover

Reply:
I need to review this

Reply:
Fixed here https://github.com/elastic/kibana/pull/170749/commits/34a33688494635e78f0cc44008e938d14b87382e